register/views.py

```diff
@@ -946,16 +946,42 @@ def landing(request):
         'is_mobile': is_mobile(request),
     }
     if request.method == "POST":
-        username = request.POST.get("username")
+        tenant = None
+        username = request.POST.get("username", '')
```

Comment:
take a look at request.POST.get("username") or ' '

Reply:
Updated, and moved username.lower inside check